src/defs/accessTokenRepresentation.ts

```diff
@@ -0,0 +1,60 @@
+import AddressClaimSet from './addressClaimSet';
+import {Category} from './resourceServerRepresentation';
+
+export default interface AccessTokenRepresentation {
+  acr?: string;
+  address?: AddressClaimSet;
+  'allowed-origins'?: string[];
+  at_hash?: string;
+  auth_time?: number;
+  authorization?: {
+    permissions: {
+      claims?: {[index: string]: string};
+      rsid?: string;
+      rsname?: string;
+      scopes?: string[];
+    }[];
+  };
+  azp?: string;
+  birthdate?: string;
+  c_hash?: string;
+  category?: Category;
+  claims_locales?: string;
+  cnf?: {
+    'x5t#S256'?: string;
```

Comment:
What a strange type, is this generated from somewhere?

Reply:
Agreed! It exists like this in the docs: https://www.keycloak.org/docs-api/11.0/rest-api/#_accesstoken-certconf

And is explicitly declared here: https://github.com/keycloak/keycloak/blob/bfce612641a70e106b20b136431f0e4046b5c37f/core/src/main/java/org/keycloak/representations/AccessToken.java#L105